src/oid.rs

```diff
@@ -157,22 +159,16 @@ impl ObjectId {
     // Represented in Big Endian.
     fn gen_count() -> [u8; 3] {
         // Init oid counter
-        if OID_COUNTER.load(Ordering::SeqCst) == 0 {
-            let start = thread_rng().gen_range(0, MAX_U24 + 1);
-            OID_COUNTER.store(start, Ordering::SeqCst);
-        }
 
         let u_counter = OID_COUNTER.fetch_add(1, Ordering::SeqCst);
 
         // Mod result instead of OID_COUNTER to prevent threading issues.
         // Static mutexes are currently unstable; once they have been
```

Comment:
Oh can you remove the comment from "Static mutexes..." and onward? It's out of date now that we're using `lazy_static`.

Reply:
fixed